skymap_scanner/server/start_scan.py

```diff
@@ -84,30 +86,14 @@ def __init__(
         self.input_pos_name = input_pos_name
         self.input_time_name = input_time_name
         self.output_particle_name = output_particle_name
-        self.reco_algo = reco_algo.lower()
-
-        # Get Position Variations
-        variation_distance = 20.*I3Units.m
-        if self.reco_algo == 'millipede_original':
-            if cfg.ENV.SKYSCAN_MINI_TEST:
-                self.pos_variations = [
-                    dataclasses.I3Position(0.,0.,0.),
-                    dataclasses.I3Position(-variation_distance,0.,0.)
-                ]
-            else:
-                self.pos_variations = [
-                    dataclasses.I3Position(0.,0.,0.),
-                    dataclasses.I3Position(-variation_distance,0.,0.),
-                    dataclasses.I3Position(variation_distance,0.,0.),
-                    dataclasses.I3Position(0.,-variation_distance,0.),
-                    dataclasses.I3Position(0., variation_distance,0.),
-                    dataclasses.I3Position(0.,0.,-variation_distance),
-                    dataclasses.I3Position(0.,0., variation_distance)
-                ]
-        else:
-            self.pos_variations = [
-                dataclasses.I3Position(0.,0.,0.),
-            ]
+        
+        self.reco_algo: str = reco_algo
```

Comment:
Do we need to persist this separately from the reco algo object? We'd save some complexity by accessing the name via an instance variable of the object

Reply:
Possibly this is now unused, or it should be. Will check.